CHANGELOG.md

```diff
@@ -1 +1,6 @@
 # InfrastructureSystems
+
+### 0.1.4
```

Comment:
What is 0.1.4?

Reply:
Just a place holder for the next version. I have seen this in other packages. Can eliminate it if you think is best.

Reply:
I claim no authority on the subject, but it seems weird to me.